src/pages/Components/Notification/index.md

```diff
@@ -3,40 +3,65 @@ title: 'Notification'
 order: 4
```

Comment:
is it the right order, we should keep all form components together

Reply:
Using `order: 9` should settle the matter

Reply:
Actually I can manage this only when this branch is merged into the main "Notification" one.